cmd/restart.go

```diff
@@ -21,6 +21,7 @@ func init() {
 
 	skipBuild = restartCmd.Flags().BoolP("skip-build", "s", false, "Skip the build phase")
 	noWatch = restartCmd.Flags().Bool("no-watch", false, "Disable autorestart")
+	noPrompt = restartCmd.Flags().BoolP("no_prompt", "n", false, "Skip confirmation prompt")
```

Comment:
Yep, i just checked in generate cmd how was handle and do the same way.

Ok for --force or -f, i'm working on it